core/modules/user/views/user.views.inc

```diff
@@ -16,12 +16,12 @@ function user_views_data() {
 
   // Define the base group of this table. Fields that don't
   // have a group defined will go into this field by default.
-  $data['users']['table']['group']  = t('User');
+  $data['users']['table']['group']  = t('User account');
 
   $data['users']['table']['base'] = array(
     'field' => 'uid',
-    'title' => t('User'),
-    'help' => t('Users who have created accounts on your site.'),
+    'title' => t('User account'),
+    'help' => t('People who have created user accounts on your site.'),
```

Comment:
this was probably fine as it was. I don't like the idea of re-introducing `people` after we worked so hard to remove that word ;)